doc/libpmemlog/pmemlog_ctl_get.3.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: manual
+Content-Style: 'text/css'
+title: _MP(PMEMLOG_CTL_GET, 3)
+collection: libpmemlog
+header: PMDK
+date: pmemlog API version 2.3
+...
+
+[comment]: <> (Copyright 2018, Intel Corporation)
+
+[comment]: <> (Redistribution and use in source and binary forms, with or without)
+[comment]: <> (modification, are permitted provided that the following conditions)
+[comment]: <> (are met:)
+[comment]: <> (    * Redistributions of source code must retain the above copyright)
+[comment]: <> (      notice, this list of conditions and the following disclaimer.)
+[comment]: <> (    * Redistributions in binary form must reproduce the above copyright)
+[comment]: <> (      notice, this list of conditions and the following disclaimer in)
+[comment]: <> (      the documentation and/or other materials provided with the)
+[comment]: <> (      distribution.)
+[comment]: <> (    * Neither the name of the copyright holder nor the names of its)
+[comment]: <> (      contributors may be used to endorse or promote products derived)
+[comment]: <> (      from this software without specific prior written permission.)
+
+[comment]: <> (THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS)
+[comment]: <> ("AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT)
+[comment]: <> (LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR)
+[comment]: <> (A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT)
+[comment]: <> (OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,)
+[comment]: <> (SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT)
+[comment]: <> (LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,)
+[comment]: <> (DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY)
+[comment]: <> (THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT)
+[comment]: <> ((INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE)
+[comment]: <> (OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.)
+
+[comment]: <> (pmemlog_ctl_get.3 -- man page for libpmemlog CTL)
+
+[NAME](#name)<br />
+[SYNOPSIS](#synopsis)<br />
+[DESCRIPTION](#description)<br />
+[CTL NAMESPACE](#ctl-namespace)<br />
+[CTL EXTERNAL CONFIGURATION](#ctl-external-configuration)<br />
+[SEE ALSO](#see-also)<br />
+
+
+# NAME #
+
+_UW(pmemlog_ctl_get),
+_UW(pmemlog_ctl_set),
+_UW(pmemlog_ctl_exec)
+-- Query and modify libpmemlog internal behavior (EXPERIMENTAL)
+
+
+# SYNOPSIS #
+
+```c
+#include <libpmemlog.h>
+
+_UWFUNCR2(int, pmemlog_ctl_get, PMEMlogpool *plp, *name, void *arg,
+	=q= (EXPERIMENTAL)=e=)
+_UWFUNCR2(int, pmemlog_ctl_set, PMEMlogpool *plp, *name, void *arg,
+	=q= (EXPERIMENTAL)=e=)
+_UWFUNCR2(int, pmemlog_ctl_exec, PMEMlogpool *plp, *name, void *arg,
+	=q= (EXPERIMENTAL)=e=)
+```
+
+_UNICODE()
+
+
+# DESCRIPTION #
```

Comment:
You should rather do separate section 5 manual for that, like @pbalcer suggest (if I understand correctly?), because now we have huge c&p for those manuals.